pages/linux/groups.md

```diff
@@ -0,0 +1,11 @@
+# groups
+
+> Print the groups a user is in.
+
+- Display for current user:
+
+`groups`
+
+- Display for another user:
```

Comment:
Perhaps `Display the groups for`..... would read better?

Reply:
Do you think "Print group memberships for users" would be better ?
(I'll also modify the doc and add an example with a list of users)

Reply:
Erm not sure here. That implies multiple users - we've got a single user here.

Reply:
Perhaps `Print group memberships for a specific user` might work instead?